Makefile

```diff
@@ -32,10 +33,18 @@ deps-update:
 
 functests: 
 	@echo "Running Functional Tests"
-	FEATURES="$(FEATURES)" hack/run-functests.sh
+	SKIP_TESTS="$(SKIP_TESTS)" FEATURES="$(FEATURES)" hack/run-functests.sh
+
+#validate is intended to validate the deployment as a whole, not focusing
+# but eventually skipping
+validateonly: 
+	@echo "Validating only"
+	SKIP_TESTS="$(SKIP_TESTS)" DONT_FOCUS=true TEST_SUITES="validationsuite" hack/run-functests.sh
```

Comment:
That means I can't validate on partial feature set?

Reply:
Yup. As the comment says, the idea when validating a "profile" is that you want to test all the features and in case you skip some (because the number of skips is expected to be very limited).

I felt this was a good compromise with the current FEATURES usage.